splinepy/microstructure/microstructure.py

```diff
@@ -559,10 +559,10 @@ def create(
                 # sensitivity evaluates to zero), perform only those that are
                 # in the support of a specific design variable
                 support = _np.where(
-                    _np.any(_np.any(tile_sensitivities != 0.0, axis=0), axis=0)
+                    _np.any(tile_sensitivities != 0.0, axis=(0, 1))
                 )[0]
                 anti_support = _np.where(
-                    _np.any(_np.all(tile_sensitivities == 0.0, axis=0), axis=0)
+                    _np.all(tile_sensitivities == 0.0, axis=(0, 1))
```

Comment:
Isn't this just negation of the mask above?

Reply:
That's true - should we handle it with an intermediate variable for the boolean list of `np.any()`, which will then be negated for the second part?

Reply:
yes

Reply:
Any updates?